<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSpriteWindow</class>
 <widget class="QDialog" name="NewSpriteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Document</string>
  </property>
  <widget class="QLabel" name="widthLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="heightLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QLabel" name="setPixelTitle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pixel Size of sprite:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="widthChanger">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>16</number>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="heightChanger">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>16</number>
   </property>
   <property name="maximum">
    <number>1024</number>
   </property>
   <property name="singleStep">
    <number>16</number>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
